NitroxServer/Communication/Packets/PacketHandler.cs

```diff
@@ -74,6 +74,10 @@ private void ProcessAuthenticated(Packet packet, Player player)
             {
                 defaultPacketProcessor.ProcessPacket(packet, player);
             }
+            if (packet.GetType().ToString() != "NitroxModel.Packets.Movement")
```

Comment:
Can you remove this?

Reply:
dont im firt time working witch GitHub need lean how make PR only files i want send